webapp/src/containers/Rhs/index.tsx

```diff
@@ -256,14 +256,14 @@ const Rhs = (): JSX.Element => {
                 </div>
             )}
             <Dialog
-                description={dialogContent}
-                destructive={showDestructivetDialog}
+                destructive={showDestructiveDialog}
                 show={showDisconnectDialog}
-                primaryActionText={showDestructivetDialog && 'Disconnect'}
+                primaryButtonText={showDestructiveDialog && 'Disconnect'}
                 onCloseHandler={() => setShowDisconnectDialog(false)}
-                onSubmitHandler={showDestructivetDialog && disconnectUser}
+                onSubmitHandler={showDestructiveDialog && disconnectUser}
                 className='disconnect-dialog'
             >
+                <p>{dialogContent}</p>
```

Comment:
Is there a specific reason we are embedding the dialog content inside a `<p>` tag?

Reply:
Since `dialogContent` is a string, so added it in `p` tag for better semantics